src/plugins/visualizations/public/legacy/__tests__/_vis.js

```diff
@@ -21,11 +21,10 @@ import _ from 'lodash';
 import ngMock from 'ng_mock';
 import expect from '@kbn/expect';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
-import { start as visualizations } from '../../legacy';
+import { npStart } from '../../../../../legacy/ui/public/new_platform';
```

Comment:
This file doesn't look right - mocha, ngMock and `ui/new_platform` in the new platform? Did this happen on purpose or should this test be converted?